# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cln/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur cln-1.1.9.orig/include/cln/string.h cln-1.1.9/include/cln/string.h
--- cln-1.1.9.orig/include/cln/string.h	2004-06-24 00:04:49.000000000 +0300
+++ cln-1.1.9/include/cln/string.h	2005-10-20 01:19:26.000000000 +0300
@@ -10,6 +10,8 @@
 
 namespace cln {
 
+struct cl_string;
+
 // General, reference counted and garbage collected strings.
 struct cl_heap_string : public cl_heap {
 private:
